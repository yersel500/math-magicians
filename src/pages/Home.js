import React from 'react';

const Home = () => {
  return (
    <>
      <h2>Welcome to our page!</h2>
      <p>This is a simple basic calculator with  functions similar to a small handheld calculator. Use this basic calculator for math with addition, subtraction, division and multiplication.
      </p>  
    
    </>
    
  );
}

export default Home;