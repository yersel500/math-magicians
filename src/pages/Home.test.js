import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Home from './Home';

it('Home page should render correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});

describe('Home Page', () => {
  test('renders App component', () => {
    render(<Home />);
    screen.debug();
  });
});
