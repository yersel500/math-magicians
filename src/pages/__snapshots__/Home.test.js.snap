// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render correctly 1`] = `
<div
  className="homepage"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    This is a simple basic calculator with  functions similar to a small handheld calculator. Use this basic calculator for math with addition, subtraction, division and multiplication.
  </p>
</div>
`;
